packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -33,6 +35,28 @@ const MAX_PROVIDERS_COUNT = 100
 const TRANSACTION_CACHE_SIZE = 50
 const BLOCK_CACHE_SIZE = 50
 
+const ABI = ['function anchor(bytes)']
+
+const iface = new Interface(ABI)
+
+//TODO finalize block number
```

Comment:
add a linear issue and reference the ticket number where this will be fixed, ie `// TODO(NET-XXXX): Finalize block number once CAS is creating smart contract anchors`